apps/project/views.py

```diff
@@ -14,7 +14,7 @@
     views,
     viewsets,
 )
-from rest_framework.decorators import detail_route, list_route
+from rest_framework.decorators import action, action
```

Comment:
only one `action` is enough I think sir.

Reply:
Fixed sir